lib/ttfunk/table/cmap.rb

```diff
@@ -8,7 +8,10 @@ def self.encode(charmap, encoding)
         result = Cmap::Subtable.encode(charmap, encoding)
 
         # pack 'version' and 'table-count'
-        result[:table] = [0, 1, result.delete(:subtable)].pack('nnA*')
+        result[:table] = EncodedString.new do |table|
```

Comment:
The block seem to be superfluous here since `<<` returns `self`. Moreover, I don't see how use of `EncodedString` helps here. We have a string and it seems perfectly fine in this case (and all the others like this).